loops-arrays-objects.js

```diff
@@ -131,5 +161,17 @@ const students = [
 // ]
 // ==========================================
 
-
-
+const neighborhoodMap = {
```

Comment:
Het gebruik van een object om postcode naar buurt te mappen is zeer efficiënt en maakt de code onderhoudsvriendelijk.